lib/Slider/Slider.js

```diff
@@ -609,6 +614,12 @@ module.exports = kind(
 		this.dragging = false;
 		Spotlight.unfreeze();
 		this.set('value',v);
+
+		if (this.enableJumpIncrement) {
+			this.$.buttonLeft.set('disabled', this.disabled || v == this.min);
+			this.$.buttonRight.set('disabled', this.disabled || v == this.max);
+		}
+
```

Comment:
Thanks for your feedback. I've tested again and it's unnecessary so I removed it.